libs/api/domains/islykill/src/lib/graphql/dto/updateIslykillSettings.input.ts

```diff
@@ -1,13 +1,17 @@
 import { Field, InputType } from '@nestjs/graphql'
-import { IsEmail, IsOptional } from 'class-validator'
+import { IsEmail, IsOptional, IsString } from 'class-validator'
 
 @InputType()
 export class UpdateIslykillSettingsInput {
   @Field(() => String)
   @IsEmail()
   email!: string
 
-  @Field(() => String, { nullable: true })
+  @Field(() => String)
+  @IsString()
```

Comment:
This btw. doesn't do anything for GraphQL.
You only need class-validators for complex field validation. Otherwise the schema will take care of itself based on its types. Same goes for IsOptional etc.

IsEmail makes sense there :+1:

Reply:
Ah! Ok thanks 👍